CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # Change Log for rocWMMA
+
 Full documentation for rocWMMA is available at [rocwmma.readthedocs.io](https://rocwmma.readthedocs.io/en/latest/).
+
+## rocWMMA 1.0 for ROCm 5.5.0
+
+### Added
+- Added support for wave32 on gfx11+
+- Added infrastructure changes to support hipRTC
+- Added performance tracking system
+
+### Changed
+- Modified the assignment of hardware information
+- Modified the data access for unsigned datatypes
+- Added library config to support multiple architectures
```

Comment:
"- Added library config to support multiple architectures" looks to belong in the Added section

Please add the following the Added section:
- Added vector cross-lane operation support

Please add the following to the Changed section:
- Refactored vector backend to be compatible with HIP_vector_type